cross/cp210x/Makefile

```diff
@@ -0,0 +1,35 @@
+PKG_NAME = cp210x
+PKG_EXT = zip
+PKG_DIST_SITE = http://www.silabs.com/Support%20Documents/Software
+PKG_DIST_NAME = Linux_3.x.x_VCP_Driver_Source.zip
+PKG_DIST_FILE = cpx210x.zip
+PKG_DIR = cpx210x
+
+REQ_KERNEL = 1
+
+HOMEPAGE = http://www.silabs.com
+COMMENT  = CP210x USB to UART Bridge VCP Drivers
+LICENSE  = 
+
+POST_EXTRACT_TARGET = myPostExtract
+CONFIGURE_TARGET = nope
+COMPILE_TARGET = myCompile
+INSTALL_TARGET = myInstall
+
+include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: myPostExtract
+myPostExtract:
+	@cd $(WORK_DIR) ; \
+	mv Linux_3.x.x_VCP_Driver_Source $(PKG_DIR) ; \
+
+.PHONY: myCompile
+myCompile:
+	cp $(KERNEL_DIR)/drivers/usb/serial/cp210x.c $(WORK_DIR)/$(PKG_DIR)/cp210x.c
+	CPPFLAGS="" CFLAGS="" LDFLAGS="" \
+	make -C $(KERNEL_DIR) SUBDIRS=$(WORK_DIR)/$(PKG_DIR) modules
+
+.PHONY: myInstall
+myInstall:
+	mkdir -p $(STAGING_INSTALL_PREFIX)/modules
+	install -m 755 $(WORK_DIR)/$(PKG_DIR)/$(PKG_NAME).ko $(STAGING_INSTALL_PREFIX)/modules/
```

Comment:
How about this? (easier for me to just drop the whole file in here in this case)
```
PKG_NAME = cp210x
PKG_DIR = $(PKG_NAME)

DOWNLOAD_TARGET = kernel #any value but `nop` for now, needs a change in spksrc.download.mk

REQ_KERNEL = 1

HOMEPAGE = http://sourceforge.net/projects/dsgpl/
COMMENT  = CP210x USB to UART Bridge VCP Drivers
LICENSE  = GNU GPL

CONFIGURE_TARGET = nope
COMPILE_TARGET = myCompile
INSTALL_TARGET = myInstall

include ../../mk/spksrc.cross-cc.mk

.PHONY: myCompile
myCompile:
	@mkdir -p $(WORK_DIR)/$(PKG_DIR)
	@echo "obj-m += cp210x.o" > $(WORK_DIR)/$(PKG_DIR)/Makefile
	@cp $(KERNEL_DIR)/drivers/usb/serial/cp210x.c $(WORK_DIR)/$(PKG_DIR)/
	$(MAKE) -C $(KERNEL_DIR) SUBDIRS=$(WORK_DIR)/$(PKG_DIR) modules

.PHONY: myInstall
myInstall:
	mkdir -p $(STAGING_INSTALL_PREFIX)/modules
	install -m 755 $(WORK_DIR)/$(PKG_DIR)/$(PKG_NAME).ko $(STAGING_INSTALL_PREFIX)/modules/
```